src/testFixtures/java/org/opensearch/neuralsearch/BaseNeuralSearchIT.java

```diff
@@ -139,7 +138,10 @@ protected String uploadModel(String requestBody) throws Exception {
         String modelGroupId = registerModelGroup();
         // model group id is dynamically generated, we need to update model update request body after group is registered
         requestBody = requestBody.replace("<MODEL_GROUP_ID>", modelGroupId);
+        return uploadModelId(requestBody);
+    }
 
+    protected String uploadModelId(String requestBody) throws Exception {
```

Comment:
What does updload model id mean?

Reply:
I already have uploadModel method signature in the BaseNeuralSearchIT. And due to file handling issues between cross modules I need to segregate it into 2 methods. So the best possible synonym I could find is uploadModelId

Reply:
I would probably call it `uploadModel(String requestBody, String groupId)` and move `requestBody = requestBody.replace("<MODEL_GROUP_ID>", modelGroupId);` into this function.

Reply:
I have made names more readable now.